migration/v0.26.x-v0.27.x/upgrade-apps.md

```diff
@@ -0,0 +1,5 @@
+# Upgrade v0.26.x to v0.27.x
+
+## Steps
+
+1. Update velero CRD's.
```

Comment:
can you add the exact steps on how to do that? I think it will make our life easier during maintenance

Reply:
done